### PR TITLE
Dts 22783 change layout hot fix3

### PR DESCRIPTION
**Pull request by brahmanand1**

### CODE REVIEW

File: UI/src/app/dashboard/filter/filter.component.css [hunk 1]
```diff
@@ -874,71 +891,203 @@ text-align: initial;
...
+  font-weight: bolder;
+  margin-left: 6px;
+}
+
+.iteration-selected-project{
+  width: 210px;
+}
+
+.iteration-selected-spring{
+  width: 225px;
```
Comment (HinPublicis) [inline UI/src/app/dashboard/filter/filter.component.css:1092]: is this width required? @brahmanand1

File: UI/src/app/dashboard/filter/filter.component.spec.ts [hunk 2]
```diff
@@ -1097,5 +1097,65 @@ describe('FilterComponent', () => {
...
+
+    it("should not redirect on User access/project access from notification for Superadmin and admin",inject([Router], (router: Router) =>{
+      const navigateSpy = spyOn(router, 'navigate');
+      spyOn(getAuthorizationService,"checkIfSuperUser").and.returnValue(true);
+      spyOn(getAuthorizationService,"checkIfProjectAdmin").and.returnValue(true);
+      component.routeForAccess("Default case");
+      expect(navigateSpy).not.toHaveBeenCalledWith(['/dashboard/Config/Profile/GrantNewUserAuthRequests']);
+    }))
+
+    it("should notification lis not null if response is comming",()=>{
```
Comment (HinPublicis) [inline UI/src/app/dashboard/filter/filter.component.spec.ts:1138]: @brahmanand1 please correct this test case statement

File: UI/src/app/dashboard/landing-page/landing-page.component.html [hunk 3]
```diff
@@ -1,134 +1,136 @@
...
+              <p-inputSwitch [(ngModel)]="checked" (onChange)="handleChange($event)">
+              </p-inputSwitch>
+              <p class="p-p-2 p-ml-2 p-mb-0 bg-yellow p-d-flex p-align-center">New projects added in the last 30 days
+                <strong class="font-big p-ml-2" *ngIf="totalProjects30Days > 0">{{totalProjects30Days}}</strong>
+              </p>
+            </div>
+            <p-table [columns]="tableHeadingArr" [value]="summaryItems">
+              <ng-template pTemplate="header" let-columns>
+                <tr>
+                  <th *ngFor="let col of columns" class="p-2 word-break bg-grey">
```
Comment (HinPublicis) [inline UI/src/app/dashboard/landing-page/landing-page.component.html:42]: @brahmanand1 please remove this commented code

File: UI/src/app/dashboard/nav/nav.component.spec.ts [hunk 4]
```diff
@@ -78,48 +81,44 @@ describe('NavComponent', () => {
...
-    expect(myFunctionSpy).toHaveBeenCalledTimes(1);
-  });
+  // it('should render message', () => {
+  //   const res = [
+  //       {
+  //           "type": "User Access Request",
+  //           "count": 0
+  //       },
+  //       {
+  //           "type": "Project Access Request",
```
Comment (HinPublicis) [inline UI/src/app/dashboard/nav/nav.component.spec.ts:112]: @brahmanand1 please remove comments

File: UI/src/app/dashboard/filter/filter.component.css (hunk 1 above)
Comment (brahmanand1) [reply UI/src/app/dashboard/filter/filter.component.css:1092]: yes , username was hiding for long sprint name

File: UI/src/app/dashboard/filter/filter.component.spec.ts (hunk 2 above)
Comment (brahmanand1) [reply UI/src/app/dashboard/filter/filter.component.spec.ts]: fixed

File: UI/src/app/dashboard/nav/nav.component.spec.ts (hunk 4 above)
Comment (brahmanand1) [reply UI/src/app/dashboard/nav/nav.component.spec.ts]: fixed

File: UI/src/app/dashboard/landing-page/landing-page.component.html (hunk 3 above)
Comment (HinPublicis) [reply UI/src/app/dashboard/landing-page/landing-page.component.html]: can still see this

Comment (brahmanand1) [reply UI/src/app/dashboard/landing-page/landing-page.component.html]: fixed, very below 3-4 line was there

**Review by HinPublicis — APPROVED** [review]

**Pull request merged**